src/entry-points/popup/App.svelte

```diff
@@ -547,13 +545,12 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
               <span>{getMessage('overTheLast', mmSs(settings.timeSavedAveragingWindowLength))}.</span>
             {/if}
           </p>
-          {#if !timeSavedOnlyOneNumberIsShown}
-            <p>{getMessage('numbersMeanings')}</p>
-          {/if}
+        <p>{getMessage('numbersMeanings')}</p>
           <ol style="padding-left: 2ch; margin-bottom: 0.25rem">
-            <li
-              style={timeSavedOnlyOneNumberIsShown ? 'list-style:none;' : ''}
-            >{timeSavedPlaybackRateEquivalents[0]} – {getMessage('timeSavedComparedToSounded')}</li>
+            <li>{timeSavedPlaybackRateEquivalents[0]} – {getMessage('timeSavedComparedToSounded')}</li>
+            {#if estimatedRemainingDuration}
+                <li>{estimatedRemainingDuration} – {getMessage('estimatedRemainingDuration')}</li>
+            {/if}
```

Comment:
I don't think it makes sense to show this in the "Number's meanings" list, because that list is supposed to tell you what the numbers that are _already displayed_ in the main UI mean.

I think this should be a separate paragraph right below this list, akin to "Equivalent time saved percentages",